spec/command/gem_helper_spec.rb

```diff
@@ -1,4 +1,5 @@
 require File.expand_path('../spec_helper', File.dirname(__FILE__))
+require 'rubygems/name_tuple'
```

Comment:
`Gem::NameTuple` is used here.
https://github.com/dbgrandi/claide-plugins/blob/26f67ca2faf4c579777d9d42134f87172614a6d1/spec/command/gem_helper_spec.rb#L28